mod/research/set/self/research_page.rb

```diff
@@ -55,11 +55,12 @@ def right_side_tabs
     tabs = {}
     if answer?
       tabs["Source"] = _render_source_tab
-      tabs["Source preview"] = _render_source_preview_tab
+      tabs["View Source"] = { content: _render_source_preview_tab,
+                              button_attr: { class: "d-none" } }
     end
```

Comment:
So we don't have these tabs at all until there's an answer, is that correct?

Reply:
that's correct